code/UserForm.php

```diff
@@ -297,12 +290,6 @@ public function getSubmissionsGridField()
             $config->addComponent(new BulkManager);
         }
 
-        Deprecation::withSuppressedNotice(function () use ($sort, $filter, $pagination) {
-            $sort->setThrowExceptionOnBadDataType(false);
-            $filter->setThrowExceptionOnBadDataType(false);
-            $pagination->setThrowExceptionOnBadDataType(false);
-        });
```

Comment:
Those are removed in https://github.com/silverstripe/silverstripe-framework/pull/11524